testutil/singlevm/setup.sh

```diff
@@ -63,6 +65,11 @@ sudo killall ciao-scheduler
 sudo killall ciao-controller
 sudo killall ciao-launcher
 sudo killall qemu-system-x86_64
+sudo killall dnsmasq
+echo "Original /etc/hosts is temporarily move to $hosts_file_backup"
```

Comment:
Yeah, I wanted to backup the hosts files. I'm working on a Ctrl + C trapper. Not a big deal, should be ready fast.